static/js/components/Toast.js

```diff
@@ -30,7 +30,7 @@ export default class Toast extends React.Component {
   render() {
     const { children, open } = this.props;
 
-    return <div className={`toast ${open ? 'open' : ''}`}>
+    return <div role="alert" className={`toast ${open ? 'open' : ''}`}>
```

Comment:
would it be a good idea to only put the `alert` role on if the toast is currently active? similarly to what we do with the `className` for it.